_posts/2017-05-17-getting-started-xml-encoding.md

```diff
@@ -0,0 +1,410 @@
+---
+layout: post
+title:  Digital text encoding and editing: Getting started
```

Comment:
No colons allowed in the metadata. I think that's the only thing messing up the build.